### DISCUSSION TITLE
Your Top Liking

### DISCUSSION THREAD

**Discussion by PRATHAM1ST** [coders-garden/coders-garden-website-nextjs#1] [Polls]
This is where we will decide which teams we want to be in while making some projects in a team. Projects will have a common goal, but people can also experiment.

**Comment by kirtankp** [comment]
@PRATHAM1ST Also Interested in NextJS with TypeScript.

**Reply by PRATHAM1ST** [reply]
Awesome, let me just add u in the team of nextjs

**Comment by Mihirthakkar1253** [comment]
@PRATHAM1ST  also interested in Simple HTML CSS JS

**Reply by PRATHAM1ST** [reply]
Adding u right away
